client/components/Trends/TrendsVisualization.vue

```diff
@@ -70,15 +74,21 @@ export default {
             datasets: [
               {
                 label: 'Cognition',
-                data: cognition
+                data: cognition, 
+                borderColor: '#008067',
+                backgroundColor: '#008067'
```

Comment:
Consider extracting these as color variables?

Reply:
i made an object earlier in the code - is that the solution you had in mind?